src/main/java/pw/kaboom/extras/modules/checks/DenyTorchLoad.java

```diff
@@ -0,0 +1,34 @@
+package pw.kaboom.extras.modules.checks;
+
+import org.bukkit.Material;
+import org.bukkit.event.EventHandler;
+import org.bukkit.event.Listener;
+import org.bukkit.event.block.BlockPlaceEvent;
+import org.bukkit.entity.Player;
+import org.bukkit.inventory.ItemStack;
+import org.bukkit.plugin.java.JavaPlugin;
+import pw.kaboom.extras.Main;
+
+public class DenyTorchLoad implements Listener {
+    private static final int limit = JavaPlugin.getPlugin(Main.class).getConfig().getInt("torchLimit");

```

Comment:
You are getting configuration values statically. This is not ideal, as it makes it impossible to change configuration values without restarting the server.